package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "fluture-retry",
+  "version": "0.0.0",
+  "description": "Toolset for retrying potentially failing computations",
+  "main": "index.js",
+  "scripts": {
+    "doctest": "sanctuary-doctest",
+    "lint": "sanctuary-lint",
+    "release": "sanctuary-release",
+    "test": "npm run lint && sanctuary-test && npm run doctest",
+    "test:unit": "sanctuary-test"
+  },
+  "keywords": [
+    "fluture",
+    "retry",
+    "async",
+    "asynchronous",
+    "functional"
+  ],
+  "files": [
+    "/index.js",
+    "/LICENSE",
+    "/package.json",
+    "/README.md"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/fluture-js/fluture-retry.git"
+  },
+  "author": "Aldwin Vlasblom <aldwin.vlasblom@gmail.com> (https://github.com/Avaq)",
+  "license": "MIT",
+  "dependencies": {},
+  "peerDependencies": {
+    "fluture": ">=6.0.0 <9.0.0"
+  },
+  "devDependencies": {
+    "codecov": "^3.0.0",
+    "fluture": "^8.0.1",
+    "sanctuary-scripts": "^1.0.1"
```

Comment:
Do you want `"^1.0.1"` or `"~1.0.1"` here?

Reply:
I believe `^` includes minor and patch versions. The way I see it: modules usually get better over time, so as long as the developer doesn't indicate a breaking change, I'm interested in the updates.

How do you view it, David?

Reply:
My habit is to use `"1.0.x"`. My reasoning is that if v1.1.0 is released with a new feature I wish to use, I'll need to change the specification to `"1.1.x"` (`"1.x.x"` would not suffice). Your approach is equally valid. In this scenario you would change the specification from `"^1.0.1"` to `"^1.1.0"`.

In summary, I should update my mental model. :)